### PR TITLE
Form submission guidance

### PR DESCRIPTION
**Pull request by davidjamesstone**

### CODE REVIEW

**Review by alexluckett — APPROVED** [review]
For now I'd hold off merging this until after refinement and we know what's happening happening with the older guidance pages. I'd bet we want markdown first to avoid dropping functionality.

File: test/form/persist-files.test.js
Comment (colinrotherham) [inline test/form/persist-files.test.js:221]: Mind if we stick to the new `@import` syntax?

```suggestion
/**
 * @import { FormMetadata, FormMetadataAuthor, FormMetadataState } from '@defra/forms-model'
 * @import { Server } from '@hapi/hapi'
 */
```

**Review by colinrotherham — APPROVED** [review]

**Review by colinrotherham — APPROVED** [review]
Approved for when the package is updated

File: test/form/persist-files.test.js
Comment (davidjamesstone) [reply test/form/persist-files.test.js]: Resolved with ca3daa7fa942f234b3f6e88d4e981b0bdec7a14a

**Pull request merged**
